doc/production/README.md

```diff
@@ -0,0 +1,43 @@
+# Production
+
+Nowadays, you can run JavaScript code in several different environments, some of them with their own particularities. Moreover, you can use `js-libp2p` for a wide range of use cases. Different environments and different use cases mean different configurations and challenges in the network.
+
+Libp2p nodes can vary from nodes behind an application, to infrastructure nodes that enable the network to operate and to be efficient. In this context, the Libp2p project provides public infrastructure to boost the network, enable nodes connectivity and improve constrained nodes performance. This public infrastructure should be leveraged for learning the concepts and experimenting. When an application on top of libp2p aims to move into production, its own infrastructure should be setup as the public nodes will be intensively used by others and its availability is not guaranteed.
+
+This guide aims to guide you from using the public infrastructure into setting up your own.
+
+## Table of Contents
+
+* [Production](#production)
+  * [Star servers](#star-servers)
+  * [Delegate nodes](#delegate-nodes)
+  * [Circuit Relay](#circuit-relay)
+  * [SSL](#ssl)
+
+## `webrtc-star` servers
+
+While the libp2p core codebase aims to work in multiple environments, there are some limitations that are not possible to overcome at the time of writing. Regarding `webRTC`, at the time of writing a set of star servers are needed to act as a rendezvous point, where peers can learn about other peers (`peer-discovery`), as well as exchange their SDP offers (signaling data).
+
+You can read on how to setup your own set of delegated nodes in [libp2p/js-libp2p-webrtc-star/DEPLOYMENT.md](https://github.com/libp2p/js-libp2p-webrtc-star/blob/master/DEPLOYMENT.md).
+
+It is worth pointing out that with new discovery protocols on the way, as well as support for distributed signaling, the star servers should be deprecated on the long run.
+
+## Delegate nodes
```

Comment:
These are IPFS only, we should clearly note that. Our only supported delegates atm leverage the IPFS http api.